src/eos-updater-poll.c

```diff
@@ -132,6 +135,15 @@ read_config (const gchar *config_file_path,
                                error))
     return FALSE;
 
+  /* FIXME: For the moment, this is undocumented and hidden. It can also be set
+   * via the PollVolume() D-Bus method. It must be non-empty if set. */
+  sources_config->override_uris = euu_config_file_get_strv (config,
+                                                            DOWNLOAD_GROUP,
+                                                            "OverrideUris", NULL,
+                                                            error);
+  if (sources_config->override_uris != NULL && sources_config->override_uris[0] == NULL)
+    g_clear_pointer (&sources_config->override_uris, g_strfreev);
```

Comment:
Not sure I understand this - why do we get the strv from the config file and then free it straight away?

Reply:
We only free it if it’s empty. These two lines basically mean we can treat (`override_uris == NULL`) and (`override_uris` is empty) as identical cases in the rest of the code, rather than having to check for both.

Reply:
I could add a comment.

Reply:
Ah, got it - I must have read this wrong.

Reply:
Pushed as a fixup.